main/getting-started/before-using-agoric.md

```diff
@@ -11,46 +12,48 @@ If you're familar with JavaScript development tools such as `node`, `yarn`, and
 ```shell
 node --version # 14.15.0 or higher
 npm install --global yarn
-git clone https://github.com/Agoric/agoric-sdk
+git clone -b community-dev https://github.com/Agoric/agoric-sdk # "community-dev" branch
```

Comment:
The scope of this PR is drifting. Please be sure to keep the title and description up to date.

Alternatively, make a separate PR regarding `-b community-dev`.